app/models/bulkrax/csv_entry.rb

```diff
@@ -32,8 +32,8 @@ def self.data_for_entry(data)
       return raw_data
     end
 
-    def self.source_identifier_field
-      Bulkrax.source_identifier_field_mapping[self.to_s] || 'source_identifier'
+    def source_identifier
+      importerexporter&.parser&.source_identifier.to_s || 'source_identifier'
```

Comment:
entry has parser directly, I think you may be taking the long way home

Reply:
yup! fixed and moved to base entry class